documentation/operators/src/nodes/nym-api.md

```diff
@@ -85,10 +136,10 @@ The API binary currently defaults to running in caching mode. You can run your A
 ./nym-api run
 ```
 
-By default the API will be trying to query a running `nyxd` process (either a validator or RPC node) on `localhost:26657`. This value can be modified either via the `--nyxd-validator ` flag on `run`:
+By default the API will be trying to query your full node running locally on `localhost:26657`. If your node is hosted elsewhere, you can specify the RPC location by using the `--nyxd-validator ` flag on `run`:
```

Comment:
maybe we should raise an issue to change the name of this flag, seems a bit misleading considering you're _not_ meant to be hitting a validator with it, but instead a fullnode

Reply:
not necessarily related to your PR feel free to ignore, was 'thinking aloud'

Reply:
Good point!

Reply:
Tracked on #4336